css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <!-- your code here 👇 -->
+<meta name="viewport" content="device-width=568, initial-scale=1">
```

Comment:
Please use `width=device-width` like in the documentation.
https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag#viewport_basics